CMakeLists.txt

```diff
@@ -39,12 +39,12 @@ set(QL_INSTALL_CMAKEDIR "lib/cmake/${PACKAGE_NAME}" CACHE STRING
     "Installation directory for CMake scripts")
 
 # Options
-option(QL_BUILD_BENCHMARK "Build benchmark" ON)
 option(QL_BUILD_EXAMPLES "Build examples" ON)
 option(QL_BUILD_TEST_SUITE "Build test suite" ON)
 option(QL_BUILD_FUZZ_TEST_SUITE "Build fuzz test suite" OFF) 
 option(QL_ENABLE_OPENMP "Detect and use OpenMP" OFF)
 option(QL_ENABLE_PARALLEL_UNIT_TEST_RUNNER "Enable the parallel unit test runner" OFF)
+option(QL_PRINT_TEST_SUITE_RUNTIMES "Print the runtimes of each test in the test suite" OFF)
```

Comment:
As a compilation flag, this is going to affect both test suite and benchmark, right?  It might make more sense to only print the test times during the benchmark run.  Maybe the timing could go in the Benchmark class?

Reply:
This flag causes the runtime of each test in the test suite to be printed.  When I was constructing the new benchmark, I needed to get some idea of how long the various tests took to run.  So I introduced this flag and the other code in the [associated commit](https://github.com/lballabio/QuantLib/pull/1962/commits/5b9f300359bc9f5cd1a1b81bbee0b21afb35f93e) so that when I ran the entire test suite, I could see how long things take.

I thought this might be generally useful to others - it may very well not be.  If it's not, then I would suggest pulling it out (and the other code in that commit).  The benchmark has its own timing code, independently of this.

What are your thoughts? Is timing individual test cases a generally useful thing?

Reply:
Timing single test cases is definitely useful (I'm doing it in [one of the CI builds](https://github.com/lballabio/QuantLib/blob/master/.github/workflows/test-times.yml) to check that we're not adding test cases that take too long) but it's also already supported by the Boost framework—if you run

    ./quantlib-test-suite --logger=JUNIT,warning:HRF,message

you'll get the usual output to console, plus an xml file with information on the single test cases.

Reply:
Ah - I didn't know this.  In which case that commit of mine serves no purpose and I'll remove it.

Reply:
I've removed the commit